src/sensai/data_transformation.py

```diff
@@ -224,37 +318,51 @@ def apply(self, df: pd.DataFrame):
                 df["%s_%d" % (columnName, i)] = encodedArray[:, i]
         return df
 
+    def info(self):
+        info = super().info()
+        info["columns"] = self._paramInfo["columns"]
```

Comment:
Consider using `info.update(self._paramInfo)` for easy extensibility

Reply:
Done